test/parallel/test-repl-deprecations.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const repl = require('repl');
+
+test();
+
+function test() {
+  testParseREPLKeyword();
+}
+
+function testParseREPLKeyword() {
+  const server = repl.start({ prompt: '> ' });
+  const warn = 'REPLServer.parseREPLKeyword() is deprecated';
+
+  common.expectWarning('DeprecationWarning', warn);
+  assert.ok(server.parseREPLKeyword('clear'));
+  common.expectWarning('DeprecationWarning', warn);
```

Comment:
I have a local change with only a single call to `common.expectWarning()`. But what I find confusing is that the tests pass whether there is one or two calls to this? Does `common.expectWarning()` not fail if the warning isn't issued?